src/main/scala/com/exasol/spark/util/Types.scala

```diff
@@ -42,7 +44,7 @@ object Types extends LazyLogging {
 
   /**
    * Maps a JDBC type [[java.sql.Types$]] to a Spark SQL [[org.apache.spark.sql.types.DataType]]
-   *
+   *Image stream 'atmoz-sftp' was marked for deletion.
```

Comment:
typo?

Reply:
@morazow yes a type.  removed.

Reply:
@jpizagno this string again somehow ended up here :)

Other than that all looks good!